oboe-talk.md

```diff
@@ -1,7 +1,7 @@
-## AJAX++ with Event-Driven JSON loading
+## A better AJAX, loading large JSON easily
 
-Working with large JSON documents in a web app can be slow and cumbersome. Don't let your UX or performance suffer! Instead bring the event driven nature of SAX to your REST data transfer, using Oboe.js. This new way of looking at AJAX makes it possible to handle dropped connections with grace, step outside the trade-off between big and small JSON, and convert any API to a mobile-friendly data source. Doing so enables faster and more reliable web apps, and increases user happiness. By the end of this talk, developers will know how to implement Oboe.js in their client and server-side code and the practical applications for doing so.
+Working with large JSON documents in a web app can be slow and cumbersome. But that is no reason to let your user experience suffer! By bringing event-driven JSON parsing to your REST data transfer, using Oboe.js, it is posible to write applications that handle large documents just as easily as small ones. This new way of looking at AJAX makes it possible to handle dropped connections with grace, step outside the trade-off between big and small JSON, and convert any API to a mobile-friendly data source. Doing so enables faster and more reliable web apps, all while making user happier. In this talk, developers will learn how to implement Oboe.js in their client and server-side code and the practical applications for doing so.
```

Comment:
It feels like "...using Oboe.js..." is just casually thrown in and dismissed. I'd suggest pulling it out of that third sentence and instead making it its own sentence introducing Oboe as the tool for this task.